src/Database/Esqueleto/Internal/Language.hs

```diff
@@ -32,6 +32,15 @@ module Database.Esqueleto.Internal.Language
   , LockingKind(..)
   , SqlString
   , ToBaseId(..)
+    -- ** Functions that work on lists
+  , orL
+  , andL
+  , andLMb
+  , whereL
+  , whereLMb
+  , onL
+  , onLMb
+  , offsetLimit
```

Comment:
I'm a little dubious on the inclusion of these functions, to be honest -- they're all somewhat trivial compositions of preexisting functions. `foldl' (&&.) (val True)` and `foldl' (||.) (val False)` are definitely things I've written in my own code before, but they don't seem quite involved enough to warrant inclusion into the library.

Reply:
The *Mb-variants are rather trivial, I agree.